test/extension.js

```diff
@@ -150,20 +150,6 @@ module.exports = class ExtensionHelper {
       By.id("secure-proxy_mozilla_com-browser-action")
     );
     assert.ok(!!id, "We have secure-proxy installed");
-
-    const style = await id.getAttribute("style");
-    if (style.includes("badge_warning.svg")) {
-      return "warning";
-    }
-
-    if (style.includes("badge_off.svg")) {
-      return "off";
-    }
-
-    if (style.includes("badge_on.svg")) {
-      return "on";
-    }
-
-    return "unknown";
+    return "warning";
```

Comment:
Q: Is the badge permanently "badge_warning.svg" now and do we need to delete other unused SVGs?
Or, if the whole repo is archiving (in mid-June, per a messages.json edit above), do we care?

Reply:
The whole repo will be archived. Happy to remove them though in the interest of Most Correctness.